static/src/stylesheets/inline/basher.ThisIsTheNHS.scss

```diff
@@ -0,0 +1,424 @@
+        z-index: 7;
+        list-style: none;
+        margin: 0;
+        padding: 0;
+    }
+
+    .grid-background {
+        background-color: #9be2de;
+        z-index: 2;
+    }
+
+    .grid,
+    .grid-background {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        right: 0;
+        width: gs-span(8) + ($gs-gutter * 2);
+        display: none;
+
+        @include mq(desktop) {
+            display: block;
+        }
+
+        @include mq(wide) {
+            width: gs-span(10) + ($gs-gutter * 2);
+        }
+    }
+
+    .hand {
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        width: auto;
+
+        @include mq(tablet, desktop) {
+            display: none;
+        }
+
+        @include mq(desktop) {
+            animation: handFade 3s cubic-bezier(.35, .5, 0, 1);
+            animation-fill-mode: forwards;
+            z-index: 4;
+        }
+    }
+
+    .hand--left {
+        right: -20px;
+        height: 104px;
+        top: 36px;
+        transform: rotate(120deg);
+
+        @include mq(desktop) {
+            transform: none;
+            height: gs-span(3);
+            top: 0;
+            right: gs-span(5);
+        }
+
+        @include mq(wide) {
+            height: auto;
+            right: gs-span(6);
+        }
+    }
+
+    .hand--right {
+        left: -44px;
+        right: 200px;
+        height: 100px;
+        transform: rotate(100deg);
+        top: 162px;
+
+        @include mq(mobileLandscape) {
+            left: -10px;
+            transform: rotate(120deg);
+        }
+
+        @include mq(desktop) {
+            transform: none;
+            height: gs-span(3);
+            left: auto;
+            right: 0;
+        }
+
+        @include mq(wide) {
+            height: auto;
+            right: -20px;
+        }
+    }
+
+    @keyframes handFade {
+        to {
+            transform: translateY(0);
+        }
+    }
+
+    .baseline {
+        background-color: $c-background;
+        position: absolute;
+        z-index: 3;
+        top: gs-height(2);
+        height: $gs-baseline;
+        width: 100%;
+
+        @for $i from 1 through 5 {
+            &.baseline--#{$i} {
+                top: gs-height($i * 2);
+            }
+        }
+    }
+
+    .gutter {
+        background-color: $c-background;
+        position: absolute;
+        z-index: 3;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        width: $gs-gutter;
+
+        @for $i from 2 through 6 {
+            &.gutter--#{$i} {
+                left: gs-span(($i * 2) - 2) + $gs-gutter;
+            }
+        }
+    }
+
+    .circle {
+        position: absolute;
+        z-index: 15;
+        right: -10px;
+        top: 180px;
+        width: 44px;
+        height: 44px;
+        border-radius: 50%;
+        background-color: $c-accent;
+        display: none;
+
+        @include mq(mobileLandscape) {
+            display: block;
+        }
+
+        @include mq(tablet) {
```

Comment:
Idea - rather than defining multiple components (`circle`, `hand`, etc.) and declaring their device-specific flavours within each block, would it perhaps make more sense to declare breakpoints and then, within them, the various changes that each entails? If I were thinking about the design, or describing it to others, that's how I'd go about organizing my description ("In mobile, we don't show the second hand, and we remove the lower circle, etc. etc.").